packages/@aws-cdk/aws-iam/lib/account-utils.ts

```diff
@@ -0,0 +1,23 @@
+import * as cdk from '@aws-cdk/core';
+
+export enum AccountCompare {
+  SAME,
+  DIFFERENT,
+  UNKNOWN,
+}
+
+export function accountsAreSameOrUnresolved(account1: string | undefined, account2: string | undefined): AccountCompare {
```

Comment:
I would be okay renaming this to `compareAccounts`, now that it returns an enum and not a boolean. What I'm saying is the question doesn't match the type of answer we're getting.